internal/app/machined/pkg/runtime/v1alpha1/bootloader/grub/testdata/grub_parse_test.cfg

```diff
@@ -20,3 +20,10 @@ menuentry "B - v2" {
   linux /B/vmlinuz cmdline B
   initrd /B/initramfs.xz
 }
+
+menuentry "Wipe system disks" {
```

Comment:
as a beginner user this might be still confusing, most users expect reset to return to maintenance mode , maybe `Reset to maintenance` still doesn't sound clear I guess

Reply:
This should't even be there -`Reset Talos installation and return to maintenance mode` is a new text. Updated PR.